utils/src/main/scala/it/pagopa/interop/commons/utils/service/impl/OffsetDateTimeSupplierImpl.scala

```diff
@@ -2,8 +2,8 @@ package it.pagopa.interop.commons.utils.service.impl
 
 import it.pagopa.interop.commons.utils.service.OffsetDateTimeSupplier
 
-import java.time.OffsetDateTime
+import java.time.{OffsetDateTime, ZoneOffset}
 
 case object OffsetDateTimeSupplierImpl extends OffsetDateTimeSupplier {
```

Comment:
Simple object?

Reply:
How about `object OffsetDateTimeSupplier extends OffsetDateTimeSupplier` ? Without the Impl. Will it work (if we don't consider the refactor needed in the codebase, ofc)?